ui.apps/src/main/content/jcr_root/apps/frontify-aem-connector/components/ffyvideo/ffyvideo.html

```diff
@@ -0,0 +1,8 @@
+<video data-sly-test="${properties.fileReference}" controls width="${properties.width}">
+    <source src="${properties.fileReference @ context='unsafe'}"
```

Comment:
Could we use attribute as a context instead?

Reply:
You're right. I was just using the same approach as the already existing image component. I've restricted urls in both components to context URI.